.gitmodules

```diff
@@ -6,4 +6,4 @@
 	url = https://github.com/moderngpu/moderngpu.git
 [submodule "cpp/thirdparty/cnmem"]
 	path = cpp/thirdparty/cnmem
-	url = https://github.com/NVIDIA/cnmem.git
+	url = https://github.com/NVIDIA/cnmem.git
```

Comment:
```suggestion
	url = https://github.com/NVIDIA/cnmem.git
	
```

Reply:
we solved the issues reported and returned to its original version when it was needed.